.github/workflows/stale.yml

```diff
@@ -13,7 +13,11 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         days-before-stale: 60
         days-before-close: 7
-        stale-issue-message: 'This issue has been marked as stale because it has been open for a while and has had no recent activity. If this issue is still important to you please drop a comment below and we will add this to our backlog to complete. Otherwise, it will be closed in 7 days.'
-        stale-issue-label: 'stale'
-        stale-pr-message: 'This PR has been marked as stale because it has been open for a while and has had no recent activity. If this PR is still important to you please drop a comment below and we will add this to our backlog to complete. Otherwise, it will be closed in 7 days.'
-        stale-pr-label: 'stale'
+        stale-issue-message: 'Hi, this issue has been open for a while and has had no recent activity so it has been labeled for attention. With significant change within our team over the past 6 months, we are no longer aware if your issue is still relevant and/or up-to-date.
```

Comment:
I think you need to do multiline yaml with a `|` -> See here https://yaml-multiline.info/

Reply:
oh nice, wasnt aware of this.. will implement in the next commit